packages/truffle-config/index.js

```diff
@@ -104,7 +108,7 @@ function Config(truffle_directory, working_directory, network) {
       transform: resolveDirectory
     },
     test_file_extension_regexp: function() {
-      return /.*\.(js|es|es6|jsx|sol)$/;
+      return /.*\.(js|ts|es|es6|jsx|sol)$/;
```

Comment:
This will just break if the user doesn't set up the typechain stuff, right?

Could we add error messaging when the user has `.ts` files, to point them in the right direction to set up everything they need?

Reply:
Ran a couple tests and it looks like we support .ts test files just fine. Will work on adding this to our automated tests to ensure coverage.